src/util.ts

```diff
@@ -166,3 +167,32 @@ export function formatAsUTCISO(
 
   return resultString;
 }
+
+export class PassThroughShim extends PassThrough {
+  private shouldEmitReading = true;
+  private shouldEmitWriting = true;
```

Comment:
Looking at this further, 'reading' and 'writing' are called on every `_read` and `_write` event - to be fully backwards-compatible this class should do the same

Reply:
Ah good catch, I misread the stream-events / stubs code. Will fix.

Reply:
Actually I do believe it is only called once: https://github.com/stephenplusplus/stubs/blob/580b4d8b3a34c1193d5ec10e0db0d6ea4b4c0ec5/index.js#L30 it looks like after the number of calls is exhausted we revert to the cached function and stream-events would stop emitting reading / writing.